### ISSUE TITLE
Export functionality as hook?

### ISSUE THREAD

**Issue by hartzis** [dogfessional/react-swipeable#113]
Look into exporting the provided functionality as a hook?
 :tada:
Thoughts/Concerns?

**Comment by hartzis** [comment]
start(ed/ing) `hooks-5.x` branch to experiment with ideas.

**Comment by hartzis** [comment]
Thoughts/Example?

The `onTouchStart, onTouchMove, onTouchEnd` event handlers are returned from calling the swipeable hook, `useSwipeable`.
```
function Swipeable () {
  const onSwipedLeft = () => console.log('Swiped Left');
  const onSwipedRight = () => console.log('Swiped Right');

  const eventHandlers = useSwipeable({ onSwipedLeft, onSwipedRight });

  return (< div {...eventHandlers} />)
}
```

**Comment by hartzis** [comment]
included in upcoming v5 #114 PR!
